demo.ipynb

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
In Step 2, we probably don't need the line `print(model_response)` since that is done in the next cell anyway

Reply:
Fixed in 08ff658.